cumulusci/cli/tests/test_cci.py

```diff
@@ -1801,31 +1801,42 @@ def test_task_doc(self, doc_task):
         run_click_command(cci.task_doc, runtime=runtime, project=False)
         doc_task.assert_called()
 
+    def test_task_doc__project__outside_project(self):
+        runtime = mock.Mock()
+        runtime.project_config = None
+        with pytest.raises(click.UsageError):
+            run_click_command(cci.task_doc, runtime=runtime, project=True)
+
     @mock.patch("click.echo")
     @mock.patch("cumulusci.cli.cci.doc_task")
     def test_task_doc_project(self, doc_task, echo):
         runtime = mock.Mock()
-        runtime.universal_config.tasks = {"test": {}}
-        runtime.project_config.config = {
-            "project": {"name": "Test"},
-            "tasks": {"option": {"a": "b"}},
-        }
-        runtime.project_config.config_project = {"tasks": {"option": {"a": "b"}}}
+        runtime.universal_config = {"tasks": {}}
+        runtime.project_config = BaseProjectConfig(
+            runtime.universal_config,
+            {
+                "project": {"name": "Test"},
+                "tasks": {"task1": {"a": "b"}, "task2": {}},
```

Comment:
Added a second task to make sure line 1377 above gets covered.

Also switched to using a real BaseProjectConfig object instead of a mock, because my code is getting `.tasks` on it, not just `.config`)